src/screens/PutawayCandidates/index.tsx

```diff
@@ -38,7 +38,15 @@ class PutawayCandidates extends Component<Props> {
     if (!this.state.refreshing) {
       const {candidates} = this.props;
       let updatedList = candidates.filter(candidate => candidate.putawayStatus === 'READY');
-      updatedList = updatedList.sort((a: any, b: any) => b['currentLocation.name'] - a['currentLocation.name']);
+      updatedList = updatedList.sort((a: any, b: any) =>
```

Comment:
Use this one:
`updatedList.sort((a: any, b: any) => a['currentLocation.name'].toLowerCase().localeCompare(b['currentLocation.name'].toLowerCase()));`
Ensure it works properly.

Reply:
not working

Reply:
@kumaransoftsuave Are you getting a specific error or anything?

Reply:
Not sorting with the code which you have shared